client/src/components/MapGeocoderWrapper.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div v-show='visible' class='geocoder-content-expanded'>
```

Comment:
I believe there should only be one root element for a component: https://eslint.vuejs.org/rules/no-multiple-template-root.html

(my IDE highlighted this for me, maybe because I have ESLint installed?)

Reply:
Done - will look into getting linter working fully. I do have some hightlighting but that was not called out.